### DISCUSSION TITLE
Getting Started - problem with loading FFmpeg library

### DISCUSSION THREAD

**Discussion by mkukawski** [SuRGeoNix/Flyleaf#464] [Q&A]
On https://github.com/SuRGeoNix/Flyleaf/wiki it says:

"NOTE 3: It is highly recommended to use Flyeaf's FFmpeg shared libraries as they are (avformat-5X.dll) patched and compiled (x64) with [FFmpeg-Builds docker](https://github.com/BtbN/FFmpeg-Builds) for a critical [HLS issue](https://patchwork.ffmpeg.org/project/ffmpeg/list/?series=1018))."

I don't understand how to use them... I downloaded .zip file from here
https://github.com/BtbN/FFmpeg-Builds/releases
https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-master-latest-win64-gpl-shared.zip
unpacked it and provided it in the Engine in the path:
C:\Users\ququ1\Downloads\ffmpeg-master-latest-win64-gpl-shared\ffmpeg-master-latest-win64-gpl-shared\
and it throws error:
"
Exception: Loading FFmpeg libraries 'C:\Users\ququ1\Downloads\ffmpeg-master-latest-win64-gpl-shared\ffmpeg-master-latest-win64-gpl-shared\' failed
"
what to do with this?

**Comment by SuRGeoNix** [comment]
Download the [Flyeaf's latest](https://github.com/SuRGeoNix/Flyleaf/releases/latest) release not from the FFmpeg-Builds.
